gravitee-apim-gateway/gravitee-apim-gateway-tests-sdk/src/main/java/io/gravitee/apim/gateway/tests/sdk/runner/GatewayRunner.java

```diff
@@ -206,6 +207,9 @@ private void configure(int gatewayPort, int technicalApiPort) throws IOException
         System.setProperty("http.port", String.valueOf(gatewayPort));
         System.setProperty("services.core.http.port", String.valueOf(technicalApiPort));
         System.setProperty("services.core.http.enabled", String.valueOf(false));
+
+        final GatewayMode mode = testInstance.getClass().getAnnotation(GatewayTest.class).mode();
+        System.setProperty("api.jupiterMode.enabled", mode.getJupiterEnabled().toString());
```

Comment:
you should put it before line 201: if a user uses the old style (with `gatewayConfigurationBuilder`) then its choice would be overridden by the default value (Jupiter)